recipes/dftd4/fake-bin/cc1

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Detecting flags from gfortran which is backed by clang fails with
```

Comment:
This hacks around the macOS issue you found which is effectively the same thing as https://github.com/conda-forge/gfortran_osx-64-feedstock/issues/11

I'll post a comment there shortly about this hack (which is hopefully enough for someone to be able to find the proper solution).

Reply:
Thanks.